app/src/main/java/org/mozilla/fenix/home/sessioncontrol/viewholders/pocket/PocketStoriesComposables.kt

```diff
@@ -73,20 +81,35 @@ fun PocketStory(
         Card(
             elevation = 6.dp,
             shape = RoundedCornerShape(4.dp),
-            modifier = Modifier
-                .size(160.dp, 87.dp)
-                .padding(bottom = 8.dp)
+            modifier = Modifier.size(160.dp, 87.dp)
         ) {
-            // Don't yet have a easy way to load URLs in Images.
-            // Default to a solid color to make it easy to appreciate dimensions
-            Box(Modifier.background(Color.Blue))
-            // Image(
-            //     painterResource(R.drawable.ic_pdd),
-            //     contentDescription = "hero image",
-            //     contentScale = ContentScale.FillHeight,
-            // )
+            ImageLoader(
+                client = client,
+                // The endpoint allows us to ask for the optimal resolution image.
+                url = story.imageUrl.replace(
+                    "{wh}",
+                    with(LocalDensity.current) {
+                        "${160.dp.toPx().roundToInt()}x${87.dp.toPx().roundToInt()}"
+                    }
+                )
```

Comment:
You could consider setting `targetSize=...` here too.

Reply:
It seems that `targetSize` (or `DesiredSize`) works on the premise that width and height are equal?
And while this may have been true for favicons for other images we need to be able to specify the two separately?
In the interim I think we can go with `maxOf(..)`.

Reply:
Went with using the width which is larger.